tjreports/tjreports.xml

```diff
@@ -4,10 +4,10 @@
 	<author>Techjoomla</author>
 	<authorEmail>extensions@techjoomla.com</authorEmail>
 	<authorUrl>https://techjoomla.com</authorUrl>
-	<copyright>Copyright (C) 2016 - 2021 Techjoomla. All rights reserved.</copyright>
+	<copyright>Copyright (C) 2016 - 2022 Techjoomla. All rights reserved.</copyright>
 	<license>http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL</license>
-	<creationDate>17th Aug 2021</creationDate>
-	<version>1.1.7</version>
+	<creationDate>25th Jan 2022</creationDate>
+	<version>2.0.0</version>
```

Comment:
@ankush-maherwal is sql file name changed to match version?

Reply:
yes, its already updated